docs/docs/domain.mdx

```diff
@@ -260,44 +288,35 @@ Make this a nice table instead.
       type: list
   ```
 
-
-
 * **Description**
 
-  The feature of this slot is set to `1` if a value with a list is set,
-  where the list is not empty. If no value is set, or the empty list is the
-  set value, the feature will be `0`. The **length of the list stored in
+  If `influence_conversation` is set to `true`, the assistant's behavior will change
+  depending on whether the list is empty or not. The **length of the list stored in
   the slot does not influence the dialogue**.
 
+#### Any Slot
 
-#### Unfeaturized Slot
+* **Type**
 
-* **Option**
-
-  `unfeaturized`
+  `any`
 
 * **Use For**
 
-  Data you want to store which shouldn't influence the dialogue flow
-
-
+  Storing arbitrary types of values.
```

Comment:
```suggestion
  Storing arbitrary values (they can be objects of any type, such as dictionaries).
```
maybe it's just me, but "type" has a special meaning in slots, and I worry that that is confusing. `type` in this case refers to any legal yaml object, not the slot types previously defined

Reply:
Is a list an object? 🤔

Reply:
or just
> they can be of any type, such as dictionaries or lists